README.md

```diff
@@ -3,28 +3,36 @@
+
+1. Okay, make sure that if you're in a private channel, `Captains Log` is installed as an app.
+2. Now, we need to make sure that you are in the chat of the channel where you want `Captains Log` to be configured in. Lets check to see if anyone on your team has already configured `/captainslog` for your current client and channel. Upon entering `/captainslog`, you should see a modal that says `Select Client for channel: <Your Channel name>`. If you do not see your client in this list, then click `x` on the top right of the modal. It is very important that you do not select anything that is not your client. If you see your client, go ahead and select it and click on the green `Submit` button. Else, lets create your client right now. Execute the command: `/captainslog new client`. Enter in the name of your client and click `Submit`. Now, go ahead and re-execute `/captainslog` and select your client.
+3. Lets create a new project. First, we need the URL for your github or gitlab repository. This can be found from copying the URL from the root of your repository's main/master branch.
+4. Go back to your slack channel from the previous step and execute the command: `/captainslog new project`. This should prompt you with a modal with 3 input fields. For the first field, select your client. For the second field, enter the name of your project. For the last field, paste in that URL that you have copied in the previous step. It is very important that the url starts with https and do not contain any query parameters. Go ahead and submit that. You should receive a message in your channel saying that new project has been added! You may repeat this step for however many projects need to be added. All releases will be handled the same for all projects under a client.
+5. Lets go ahead and configure webhooks for your repository. In github, you're going to need to be a repository admin. If you are not an admin, have an admin complete this step. From your repository, navigate to `Settings > Webhooks > Add webhook`. From here, enter in the endpoint: `https://captains-log-dev.fly.dev/webhook`, then click `Let me select individual events`. Scroll down, unselect `Pushes` and select `Releases`. Then scroll down and click the `Add webhook` button. If there are multiple projects with different repositories, then this step will need to be applied to each repository.
+6. That's it! Your Slack channel should get release notifications as soon as they are released in github. If this is not the case, you may reach out to `@zachbob` on Slack for some assistance.
+7. Once your repository is configured, all release notes and release events are posted in this [Confluence Directory](https://mojotech.atlassian.net/wiki/spaces/PM/pages/29458448/The+Captain).
+8. If you wish to manually create release notes, this can be done by entering this `/captainslog` command and filling out the modal form.
```

Comment:
maybe add some phrasing to let users know this is not connected to a github release and will simply post whatever entered in the modal to confluence under the same configured parent page.